iso690-full-note-en.csl

```diff
@@ -0,0 +1,616 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>ISO-690 (full note, English)</title>
+    <id>http://www.zotero.org/styles/iso690-full-note-en</id>
+    <link href="http://www.zotero.org/styles/iso690-full-note-en" rel="self"/>
+    <link href="http://www.zotero.org/styles/iso690-full-note-sk" rel="template"/>
+    <link href="https://www.citace.com/CSN-ISO-690.pdf" rel="documentation"/>
+    <author>
+      <name>Oldrich Tristan Florian</name>
+      <email>oldrich.florian@gmail.com</email>
+      <uri>http://otristan.com</uri>
+    </author>
+    <author>
+      <name>Martin Košatka</name>
+      <email>kosatka.martin@gmail.com</email>
+      <uri>http://github.com/elladan1337</uri>
+    </author>
+    <category citation-format="note"/>
+    <category field="law"/>
+    <summary>English ISO-690, full note.</summary>
+    <updated>2021-03-07T01:38:13+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
```

Comment:
I know these are in some other styles, but could you remove the ones that are identical to the local (in, online, from, accessed), and also remove no date, adding the square brackets in the style rather than the locale.